docker/k8s/fabric/README.md

```diff
@@ -0,0 +1,59 @@
+# Kubernetes playground for Avalon Fabric proxy model
+
+This serves as a playground to setup k8s cluster for Avalon Fabric proxy model. 
+
+## Prerequisites
+- To setup the k8s cluster using minikube, refer to Prerequisites section in README.md under docker/k8s folder
+- Build docker images by running the below command
+  `docker-compose -f docker-compose-fabric.yaml build`
+- Start fabric network.
+  ```bash
+   # Go to avalon root directory and run the below command
+   ./scripts/start_fabric.sh -u
+  ```
+
+## Running 
+
+1. Start Shared KV database service.
+    ```bash
+    # bootstrap the deployment and service
+    kubectl create -f ../lmdb.yaml
+    ```
+2. Start `EnclaveManager` (to manage Intel SGX enclaves)  
+    ```bash
+    kubectl create -f ../enclave-manager-deployment.yaml
```

Comment:
This deployment file is for SGX SIM. For SGX HW we need `kubectl create -f ../sgx-enclave-manager-deployment.yaml`. Can we add this step for Intel SGX hardware mode.

Reply:
For fabric proxy, I referred to docker-compose-fabric.yaml which is only for SGX SIM. If fabric proxy is running well with SGX HW then yes, what you are asking for, it can be done.

Reply:
Technically it should work as enclave manager is independent of direct/proxy model. But we should test this without fail.

Reply:
Yes, it works.